proto/datastream/v1/datastream.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package datastream.v1;
+
+option go_package = "github.com/0xPolygonHermez/zkevm-node/state/datastream";
+
+message Batch {
```

Comment:
Since we are modifying the datastream....I would add a field here: `isForced`
Because, forced batches use some data in a different manner so it will be useful to know if a batch is forced.
Right now, forced batches are not enabled...but injected are

Reply:
discussed AFK: `batchType` could fit better here and it also could be useful in the future